unlock-wordpress-plugin/inc/classes/class-fullpostpage.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Registers assets for full post/page functionality.
+ *
+ * @package unlock-protocol
+ */
+
+namespace Unlock_Protocol\Inc;
+
+use Unlock_Protocol\Inc\Fullpostpage\Unlock_Box_Fullpp;
+use Unlock_Protocol\Inc\Traits\Singleton;
+
+/**
+ * Class FullPostPage
+ *
+ * @since 3.0.0
+ */
+class Fullpostpage {
```

Comment:
```suggestion
class FullPostPage {
```

Reply:
okay done. Though had to sacrifice not been able to change some other names to implement this due to the Singleton design pattern traits.